android/demos/rib-workers/src/main/kotlin/com/uber/rib/workers/root/main/workers/UiWorker.kt

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (C) 2023. Uber Technologies
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.uber.rib.workers.root.main.workers
+
+import com.uber.rib.core.RibDispatchers
+import com.uber.rib.core.Worker
+import kotlin.coroutines.CoroutineContext
+
+class UiWorker : Worker {
```

Comment:
Good overall, but it still leaves the concept of a worker as pretty ambiguous, with most of these being empty or just a sleep. We should try to flesh these out a bit more to teach the concept.

Reply:
Makes sense, will add more details for this type of workers. E.g. Io Worker for some file manipulation, etc